docs/build-your-first-blueprint.md

```diff
@@ -1,6 +1,6 @@
 ## Build your first Blueprint
 
-Let's build a simple Blueprint that
```

Comment:
There are a lot of "B's" up here - do you think we can use "simple" for readability?

Reply:
Haha, I like alliterations. How about 'elementary' or 'straightforward'

Reply:
> Haha, I like alliterations. How about 'elementary' or 'straightforward'

commented when I saw the change, absolutely love 'elementary'